src/__tests__/annotations.test.ts

```diff
@@ -281,27 +282,41 @@ describe('Annotations Query', () => {
         query: "events{assetIds=[$AssetVariable], description!~'event.*'}",
       },
     };
-    const response = _.cloneDeep(annotationResponse);
-    response.data.items = annotationResponse.data.items.filter(item =>
+    const annotationOption2: any = {
+      ...annotationOption1,
+      annotation: {
+        query: 'events{assetIds=[${MultiValue:csv}]}',
+      },
+    };
+    const response1 = _.cloneDeep(annotationResponse);
+    const response2 = _.cloneDeep(annotationResponse);
```

Comment:
it doesn't seem it's being used.

Reply:
ah.. I just didn't finish with those tests. Will fix